--- a/luacom.cpp
+++ b/luacom.cpp
@@ -2656,6 +2656,7 @@
   lua_pushboolean(L, 0);
   luaCompat_moduleSet(L, MODULENAME, LUACOM_SHOULD_ABORT_API);
 
+#if 0 // MUSHclient controls loading of Lua helper code.
   // loadls the lua code that implements the remaining
   // features of LuaCOM
   int status = 0;
@@ -2674,6 +2675,7 @@
     fprintf(stderr, "luacom.dll error: %s\n", msg ? msg : "unknown");
     lua_error(L);
   }
+#endif
 
   idxDispatch = (void*)&luacom_runningInprocess;
 
